utilities/shared_disk.py

```
@@ -97,48 +102,107 @@ def _write_marker(ssh_conn: VMSSHConnection, file_path: str, content: str, vm_la
     _run_cmd_on_vm(ssh_conn, ["sudo", "sync"], f"{vm_label} sync")
 
 
-def _get_shared_disk_device(prepared_plan: dict[str, Any], vm_name: str) -> str:
-    """Determine the shared disk device path from source VM disk layout.
+def _get_pvc_names_from_vm(
+    ocp_admin_client: "DynamicClient",
```

Ruff (0.15.15)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)